sys/xtimer/Kconfig

```diff
@@ -7,7 +7,7 @@
 
 menuconfig MODULE_XTIMER
     bool "xtimer"
-    depends on HAS_PERIPH_TIMER || MODULE_XTIMER_ON_ZTIMER || MODULE_ZTIMER_XTIMER_COMPAT
+    depends on HAS_PERIPH_TIMER
```

Comment:
Note that `ZTIMER_USEC` could be provided by other timers than `periph_timer` as well. E.g. the `periph_ptp` would be one option, but it should be possible (even though likely not too sensible) to also run `ZTIMER_USEC` on top of an RTT. (Some applications will not behave well when the resolution of `ZTIMER_USEC` is > 30 us, though.)

If there is an easy fix to not depend on `periph_timer` when `ztimer` is as backend, this would be nice. Otherwise, I think this should get in as is and we should focus on getting rid of `xtimer` for good.

Reply:
Hmm I see, ok I'll try to look into this, but I'm suspecting such behaviour could end up again in a loop.

Reply:
This is really a niche scenario and `xtimer` is not going to stay, anyway. So I wouldn't spend more than 5 minutes on this, if I were in your shoes ;-)